tests/system/Storage/RequesterPaysTest.php

```diff
@@ -210,4 +343,19 @@ function (Bucket $bucket, StorageObject $object) {
             ]
         ];
     }
+
+    public function testDeleteNotification()
+    {
+        $bucket = $this->requesterPaysClient->bucket(self::$bucketName, $this->requesterProject);
+        $bucket->notification(self::$notificationId)->delete();
```

Comment:
Apologies, should have noted this one as well previously. An assertion here would be great as well.

Reply:
the @expectedException notation is counted as an assertion.

Reply:
Yeah, but isn't that on the test below?

Reply:
ack, sorry. yep. fixed now